website/content/docs/nia/enterprise/index.mdx

```diff
@@ -7,8 +7,21 @@ description: >-
 
 # Consul-Terraform-Sync Enterprise
 
-Consul-Terraform-Sync Enterprise features address organization complexities of collaboration, operations, scale, and governance. Please see the sidebar navigation on the left to view all enterprise features.
+Consul-Terraform-Sync (CTS) Enterprise is available with [Consul Enterprise](https://www.hashicorp.com/products/consul) and requires a Consul [license](/docs/nia/enterprise/license) to be applied.
 
-Enterprise features require a Consul [license](/docs/nia/enterprise/license) to be applied.
+Enterprise features of CTS address organization complexities of collaboration, operations, scale, and governance. CTS Enterprise supports an official integration with [Terraform Cloud](https://www.terraform.io/cloud) to extend insight into dynamic updates of your network infrastructure.
 
-These features are part of Consul-Terraform-Sync Enterprise which is part of [Consul Enterprise](https://www.hashicorp.com/products/consul).
+| Features | Open Source | Enterprise |
+|----------|-------------|------------|
+| | Self-Managed | Self-Managed |
+| Consul Namespace | Default namespace | Filter task triggers by namespace |
+| Network Automation | Terraform OSS | Terraform OSS or Terraform Cloud |
+| Terraform Workspaces | Local | Local or [remote workspaces](https://www.terraform.io/cloud-docs/workspaces) which include run [history](https://www.terraform.io/docs/cloud/run/manage.html), [triggers](https://www.terraform.io/docs/cloud/workspaces/run-triggers.html) and [notifications](https://www.terraform.io/docs/cloud/workspaces/notifications.html) |
+| Terraform Backend Options | [azurerm](https://www.terraform.io/docs/backends/types/azurerm.html), [consul](https://www.terraform.io/docs/backends/types/consul.html), [cos](https://www.terraform.io/docs/backends/types/cos.html), [gcs](https://www.terraform.io/docs/backends/types/gcs.html), [kubernetes](https://www.terraform.io/docs/backends/types/kubernetes.html), [local](https://www.terraform.io/docs/backends/types/local.html), [manta](https://www.terraform.io/docs/backends/types/manta.html), [pg](https://www.terraform.io/docs/backends/types/pg.html), [s3](https://www.terraform.io/docs/backends/types/s3.html) | The supported backends or Terraform Cloud |
+| Terraform Version | One Terraform version for all tasks | Optional Terraform version per task when using the Terraform Cloud driver |
+| Terraform Run Output | CTS logs | CTS logs or Terraform output organized by Terraform Cloud remote workspaces |
+| Audit | | Terraform audit logs ([Terraform Cloud](https://www.terraform.io/cloud-docs/api-docs/audit-trails) or [Terraform Enterprise](https://www.terraform.io/enterprise/admin/infrastructure/logging)) |
```

Comment:
Maybe a "None" or "No built in support" or something along those lines rather than a blank entry? 🤔  Don't have a super strong opinion, though, was just my initial thought.

Reply:
I would prefer a blank entry so it's visually clear the difference in features. We can revisit once we have more Enterprise only features.